apps/landing/components/Terms/Content.tsx

```diff
@@ -0,0 +1,394 @@
+          {intl("terms/body/privacy-and-protection-content")}
+        </Typography>
+      </div>
+
+      <div className="flex flex-col gap-4 sm:gap-6">
+        <Typography
+          tag="h2"
+          weight="bold"
+          className="text-natural-950 text-[1.5rem] sm:text-[2rem]"
+        >
+          {intl("terms/body/payment-and-subscription-title")}
+        </Typography>
+
+        <div className="flex flex-col gap-2">
+          <li className="mr-6 text-brand-500 text-[1.5rem]">
+            <Typography
+              tag="p"
+              weight="medium"
+              className="text-natural-950 text-[1rem] sm:text-[1.5rem]"
+            >
+              {intl("terms/body/payment-and-subscription-content/1")}
+            </Typography>
+          </li>
+
+          <li className="mr-6 text-brand-500 text-[1.5rem]">
+            <Typography
+              tag="p"
+              weight="medium"
+              className="text-natural-950 text-[1rem] sm:text-[1.5rem]"
+            >
+              {intl("terms/body/payment-and-subscription-content/2")}
+            </Typography>
+          </li>
+
+          <li className="mr-6 text-brand-500 text-[1.5rem]">
+            <Typography
+              tag="p"
+              weight="medium"
+              className="text-natural-950 text-[1rem] sm:text-[1.5rem]"
+            >
+              {intl("terms/body/payment-and-subscription-content/3")}
+            </Typography>
+          </li>
+
+          <li className="mr-6 text-brand-500 text-[1.5rem]">
+            <Typography
+              tag="p"
+              weight="medium"
+              className="text-natural-950 text-[1rem] sm:text-[1.5rem]"
+            >
+              {intl("terms/body/payment-and-subscription-content/4")}
+            </Typography>
+          </li>
+        </div>
+      </div>
+
+      <div className="flex flex-col gap-4 sm:gap-6">
+        <Typography
+          tag="h2"
+          weight="bold"
+          className="text-natural-950 text-[1.5rem] sm:text-[2rem]"
+        >
+          {intl("terms/body/refund-policy-title")}
+        </Typography>
+        <Typography
+          tag="p"
+          weight="medium"
+          className="text-natural-950 text-[1rem] sm:text-[1.5rem]"
+        >
+          {intl("terms/body/refund-policy-content")}
+        </Typography>
+      </div>
+
+      <div className="flex flex-col gap-4 sm:gap-6">
+        <Typography
+          tag="h2"
+          weight="bold"
+          className="text-natural-950 text-[1.5rem] sm:text-[2rem]"
+        >
+          {intl("terms/body/booking-steps-title")}
+        </Typography>
+
+        <ol className="flex flex-col gap-2 text-brand-500">
+          <li className="text-brand-500 text-[1.5rem]">
+            <Typography
+              tag="p"
+              weight="medium"
+              className="text-natural-950 text-[1rem] sm:text-[1.5rem]"
+            >
+              1. {intl("terms/body/booking-steps-content/1")}
```

Comment:
The numbers should come from the list styling. Check [tailwind](https://tailwindcss.com/docs/list-style-type) .